todo_app/app.py

```diff
@@ -1,9 +1,9 @@
 from flask import Flask, redirect, render_template, request
-from todo_app.data.trello_items import add_item, complete_item, get_items
-
+from todo_app.data.cosmosdb_items import add_item, get_items, complete_item
```

Comment:
Nice; the fact this is such a clean change here suggests your interface was well chosen & implemented earlier!